README.md

```diff
@@ -1,8 +1,8 @@
-# attribute-service
+# Attribute-service
```

Comment:
nit: upper case A followed by a `-service` doesn't feel right

Reply:
Okay! will make it `Attribute service`

Reply:
make it match deployments "attribute-service" because this is literally something people will see in logs etc